feeluown/gui/helpers.py

```diff
@@ -132,6 +132,8 @@ def setModel(self, model):
         super().setModel(model)
         model.rowsInserted.connect(self.on_rows_changed)
         model.rowsRemoved.connect(self.on_rows_changed)
+        if 'no_more_item' in dir(model.sourceModel()):
+            model.sourceModel().no_more_item.connect(self.on_rows_changed)
```

Comment:
Too tricky 🌚

Reply:
When the items are fetched, the rows are inserted and `rowsInserted` signal should be emitted.